### ISSUE TITLE
how to pass a model from training phase to test phase

### ISSUE THREAD

**Issue by eyaler** [ppwwyyxx/tensorpack#188]
I want to train a model, and then evaluate on the test set. is there a better way than writing code to look for the last saved model file?

**Comment by ppwwyyxx** [comment]
I'm not sure what you mean exactly.
But if you want to load the latest model in a separate script, "LOGGER_DIR/checkpoint" is the last saved checkpoint.

**Comment by eyaler** [comment]
sorry if i was unclear. i wanted to get the model_file after training and pass to sample(). i ended up using:

`model_file = tf.train.get_checkpoint_state(logger.LOG_DIR).model_checkpoint_path
 sample(args.data, model_file)`

**Comment by ppwwyyxx** [comment]
This would depend on how "sample" is written.
If it uses `SaverRestore(model_file)` in the end, then model_file can either be `dir/checkpoint`, or what you have right now will also work.

**Comment by eyaler** [comment]
so:

model_file = os.path.join(logger.LOG_DIR, 'checkpoint')

**Comment by ppwwyyxx** [comment]
yes.
